test/saithrift/tests/sail3.py

```diff
@@ -184,6 +187,77 @@ def runTest(self):
 
             self.client.sai_thrift_remove_virtual_router(vr_id)
 
+def ip6_to_integer(ip6):
```

Comment:
To do that will need us to install that dependency in the ptf client docker. Will apply that change as an independent PR after finish the docker modification.